core/src/main/scala/kafka/zookeeper/ZooKeeperClient.scala

```diff
@@ -477,6 +448,26 @@ trait ZNodeChildChangeHandler {
   def handleChildChange(): Unit = {}
 }
 
+// Thin wrapper for zookeeper.Op
+sealed trait ZkOp {
+  def toZookeeperOp: Op
+}
+
+case class CreateOp(path: String, data: Array[Byte], acl: Seq[ACL], createMode: CreateMode) extends ZkOp {
+  override def toZookeeperOp: Op = Op.create(path, data, acl.asJava, createMode)
+}
```

Comment:
nits: can you add space between `case class` to be consistent with the existing code style?

Reply:
Done.